### PR TITLE
Add missing acceptance tests back

### PR DESCRIPTION
**Pull request by pepibumur**
### Short description 📝

The following acceptance tests were not running on CI:
- `generate-6`.
- `precompiled`.
- `env`
- `doc`

This PR adds them back and makes sure they run successfully.

### CODE REVIEW

**Review by RomainBoulay — APPROVED** [review]
Thanks tio

File: .github/workflows/tuist.yml [hunk 1]
```diff
@@ -78,13 +62,15 @@ jobs:
...
             'generate-5',
+            'generate-6',
             'init',
             'lint-project',
             'lint-code',
             'scaffold',
             'up',
             'build',
             'test',
+            'env',
```
Comment (fortmarek) [inline .github/workflows/tuist.yml:73]: Should not `doc` added be here, too?

**Review by fortmarek — APPROVED** [review]
Looks good, thanks 👏

File: .github/workflows/tuist.yml (hunk 1 above)
Comment (pepibumur) [reply .github/workflows/tuist.yml:73]: I removed `doc` from now from the test-suite until I can think of a way to implement it without any inherent flakiness.

**Pull request merged**